packages/plugin-legacy/src/index.ts

```diff
@@ -119,7 +110,7 @@ const _require = createRequire(import.meta.url)
 
 function viteLegacyPlugin(options: Options = {}): Plugin[] {
   let config: ResolvedConfig
-  const targets = options.targets || 'defaults'
+  const targets = options.targets
```

Comment:
In the last meeting, we discussed this one. We think `browserslist` should have precedence over the default value. Also the current default value is not ideal, given that `'defaults'` is not so different from Vite's (non-legacy) default browser target and plugin-legacy users would use a lower target than `'defaults'` usually.

I think we could include the default value change in this PR. But if you are not interested in investigating about that, we can go with this PR as-is. I'll create follow up PR in that case.

Reply:
I wonder that what value will be set. In my opinions, the value `undefined` is most appropriate default value, because any value than `undefined` will stop detecting `browserslist`.
I have updated documentation about targets, if you want to modify default `targets`, take control of it.

Reply:
> because any value than `undefined` will stop detecting `browserslist`.

Yes, so we need to call `browserslist` on our side first before calling babel.

Reply:
OK, I will do this, and which default value do you want to add?

Reply:
> which default value do you want to add?

We need to think about this. I haven't thought what is the best yet.

Reply:
Sorry for taking time. I've been thinking about the new default target.
[`defaults`](https://browsersl.ist/#q=defaults) is a similar target to our target for non-legacy (`es2020,edge88,firefox78,chrome87,safari14`) but I noticed that `defaults` includes "Opera Mini" (uses Opera 12.1 internally and [doesn't completely support ES5](https://caniuse.com/es5)) and the result will be very different.

So maybe we could go with the current default (`defaults`).

Or we could widen the target a bit more to [`> 0.1%, last 2 versions, Firefox ESR`](https://browsersl.ist/#q=%3E+0.1%25%2C+last+2+versions%2C+Firefox+ESR) (this covers 93.9 %). `defaults` is an alias of `> 0.5%, last 2 versions, Firefox ESR, not dead` and covers 88.9%.

@patak-dev @ArnaudBarre What do you think?

FYI [swc](https://swc.rs/docs/configuration/supported-browsers#targets) tries browserslist and fallbacks to `{}` (transpile everything) which is same with [babel](https://babeljs.io/docs/en/options#targets).

Reply:
No opinion at all on this, I don't work on any application that need legacy support. But from my experience, if ie11 is included in the default that good enough, people who want more precise support will probably provide a list

Reply:
[`last 2 versions and not dead, > 0.3%, Firefox ESR`](https://browsersl.ist/#q=last+2+versions+and+not+dead%2C+%3E+0.3%25%2C+Firefox+ESR) seems to be a good one to choose. This includes IE 11 and iOS 12.x that is still maintained a bit.
Let's go with this one.